policy/common/actor.go

```diff
@@ -45,6 +45,10 @@ func (a *Actors) IsEmpty() bool {
 		len(a.Permissions) == 0 && !a.Admins && !a.WriteCollaborators)
 }
 
+func (a *Actors) IsZero() bool {
+	return a.IsEmpty()
+}
```

Comment:
I think it would also be fine to rename the `IsEmpty` method to `IsZero` here

Reply:
Got it, cheers!